### PR TITLE
[FLINK-21176][docs] translate the updates of 'avro-confluent.zh.md' to Chinese

### PR DESCRIPTION
**Pull request by jjiey**
## What is the purpose of the change

[FLINK-21176](https://issues.apache.org/jira/browse/FLINK-21176)
Translate the updates of 'avro-confluent.zh.md' to Chinese

## Brief change log

  - Translate the updates to Chinese, doc location is 'docs/dev/table/connectors/formats/avro-confluent.zh.md'

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? not documented

### CODE REVIEW

**Comment by jjiey** [comment]
Hi, @wuchong , could you help to review this? thanks.

**Comment by wuchong** [comment]
@fsk119  could you help to review this?

**Comment by sjwiesman** [comment]
Hi @jjey 

Thank you for working on this translation! Can you please rebase this change on master, we recently migrated Flinks documentation to Hugo from Jekyll. You can find full details on building the new documentation locally, along with our provided "shortcodes" in `docs/README.md`

**Comment by jjiey** [comment]
Hi @sjwiesman 

It's done already. Thanks for your suggestion.

File: docs/content.zh/docs/connectors/table/formats/avro-confluent.md [hunk 1]
```diff
@@ -115,41 +115,40 @@ CREATE TABLE user_created (
...
-  -- subjects have a default value since Flink 1.13, though can be overriden:
+  -- 自 Flink 1.13 起，subjects 具有一个默认值, 但是可以被覆盖：
   'key.avro-confluent.schema-registry.subject' = 'user_events_example2-key2',
   'value.avro-confluent.schema-registry.subject' = 'user_events_example2-value2'
 )
 ```
 
 ---
-Example of a table using the upsert connector with the Kafka value registered as an Avro record in the Schema Registry:
+使用 upsert 连接器，Kafka 的 value 在 Schema Registry 中注册为 Avro 记录的表的示例：
```
Comment (fsk119) [inline docs/content.zh/docs/connectors/table/formats/avro-confluent.md:138]: I think it's better to use fully quailfied name `upsert-kafka`. Only upsert may confuse users.

**Review by fsk119 — COMMENTED** [review]
Sorry for the late response. LGTM except one minor suggestions.

File: docs/content.zh/docs/connectors/table/formats/avro-confluent.md (hunk 1 above)
Comment (jjiey) [reply docs/content.zh/docs/connectors/table/formats/avro-confluent.md]: Yes, I agree. Thank for your suggestion.

**Comment by jjiey** [comment]
Thanks @fsk119 for your review comments. I've updated and commited the modifications. 
And Happy Niu Year to @fsk119 and @wuchong.

File: docs/content.zh/docs/connectors/table/formats/avro-confluent.md [hunk 2]
```diff
@@ -115,41 +115,40 @@ CREATE TABLE user_created (
...
   kafka_key_id STRING,
   
-  -- a few columns mapped to the Avro fields of the Kafka value
+  -- 映射到 Kafka value 中的 Avro 字段的一些列
   id STRING, 
   name STRING, 
   email STRING, 
   
-  -- upsert-kafka connector requires a primary key to define the upsert behavior
+  -- upsert-kafka 连接器需要一个主键来定义 upsert 行为
```
Comment (fsk119) [inline docs/content.zh/docs/connectors/table/formats/avro-confluent.md:151]: Could you help to fix this line doc in English?

**Review by fsk119 — APPROVED** [review]
Thanks for  your wishes! Happy niu year =-=

It would be fine if you could fix the last minor comments.

File: docs/content.zh/docs/connectors/table/formats/avro-confluent.md (hunk 2 above)
Comment (jjiey) [reply docs/content.zh/docs/connectors/table/formats/avro-confluent.md:151]: I guess you don't mean to fix this line, maybe the line 132 of the English doc：'Example of a table using the upsert connector with the Kafka value registered as an Avro record in the Schema Registry:', I should fix 'upsert connector' to 'upsert-kafka connector', right？

Comment (fsk119) [reply docs/content.zh/docs/connectors/table/formats/avro-confluent.md:151]: Yes. You are right.

**Comment by fsk119** [comment]
Thanks for your update @jjiey .  Cheers!

CC @wuchong

**Comment by jjiey** [comment]
Cheers! Thanks @fsk119 for your guidance!

**Review by wuchong — APPROVED** [review]
LGTM.

**Pull request merged**